application/uitests/org.openjdk.jmc.test.jemmy/src/main/java/org/openjdk/jmc/test/jemmy/misc/wrappers/JfrWizard.java

```diff
@@ -446,6 +455,22 @@ private boolean setEventSetting(Boolean desiredState, String settingName, String
 		return currentState;
 	}
 
+	public void setEventSettingInvertOthers(boolean desiredState, String settingName, String ... path) {
+		// invert everything
+		// the actual names are observed to always at least contain the roots of the tree
+		// and setting them also sets all their children
+		{
+			final var eventSettingsTree = getEventSettingsTree();
+			final var names = eventSettingsTree.getAllItemTexts();
+			for (var name : names) {
+				final var currentPath = name.toArray(String[]::new);
+				setEventSetting(!desiredState, settingName, currentPath);
```

Comment:
Also here: `setEventSetting(!desiredState, settingName, name.toArray(String[]::new));`

Reply:
I mean yes but I would question that it is clearly "better".